test/common/upstream/BUILD

```diff
@@ -468,8 +468,8 @@ envoy_cc_test(
     srcs = ["bounded_load_hlb_test.cc"],
     deps = [
         ":utility_lib",
-        "//source/common/upstream:thread_aware_lb_lib",
         "//source/common/upstream:upstream_includes",
+        "//source/extensions/load_balancing_policies/common:thread_aware_lb_lib",
```

Comment:
Will it make sense to move this test to the extensions? IIUC it will also fix the need to have the extra-visibility for thread_aware_lb_lib.